docs/tutorial/index.md

```diff
@@ -4,7 +4,7 @@ title: Gatsby.js Tutorial
 
 Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
 
-The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/) page.
+> The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/) page.
```

Comment:
👍

We should consider styling blockquotes (with a certain class? all?) like [reactjs.org does them](https://reactjs.org/tutorial/tutorial.html). I quite like them!

![screen shot 2018-11-13 at 10 12 02 am](https://user-images.githubusercontent.com/3924690/48426551-a2296800-e72c-11e8-9111-7ee87f36fbd9.png)